config/LinkConfigTitan.properties

```diff
@@ -14,7 +14,7 @@
 # Path for workload properties file.  Properties in this file will override
 # those in workload properties file.
 # Can be absolute path, or relative path from LinkBench home directory
-workload_file = config/FBWorkload.properties
+workload_file = config/FBWorkloadTest.properties
```

Comment:
Maybe not commit changes to the config file that are not default settings; you can always pass commands to linkbench  executable like so:

```bash
./bin/linkbench -Dworkload_file=config/FBWorkloadTest.properties ...
```